pkg/common/cns-lib/volume/manager.go

```diff
@@ -2260,3 +2262,27 @@ func (m *defaultManager) DeleteSnapshot(ctx context.Context, volumeID string, sn
 	}
 	return err
 }
+
+// ProtectVolumeFromVMDeletion helps set FCD_KEEP_AFTER_DELETE_VM control flag for given volumeID
+func (m *defaultManager) ProtectVolumeFromVMDeletion(ctx context.Context, volumeID string) error {
```

Comment:
Can we add unit tests for this method

Reply:
I can take this later. We do not have framework for unit testing this function `ProtectVolumeFromVMDeletion` and other methods for the VolumeMigrationService interface.